src/model/__tests__/MediaDataSource.ts

```diff
@@ -1,143 +1,166 @@
 import mongoose from 'mongoose'
-import muuid from 'uuid-mongodb'
+import muuid, { MUUID } from 'uuid-mongodb'
 import MutableMediaDataSource from '../MutableMediaDataSource'
 import AreaDataSource from '../MutableAreaDataSource'
+import ClimbDataSource from '../MutableClimbDataSource'
 
-import { connectDB, createIndexes, getAreaModel, getMediaModel } from '../../db/index.js'
-import { MediaInputType, TagEntryResultType } from '../../db/MediaTypes.js'
+import { connectDB, createIndexes } from '../../db/index.js'
 import { AreaType } from '../../db/AreaTypes.js'
+import { EntityTag, MediaObject, MediaObjectGQLInput, AddTagEntityInput } from '../../db/MediaObjectTypes.js'
```

Comment:
I've been favoring writing integration tests so that we can validate and document the graphQL layer as well rather than just data sources down. They are in `src/__tests__/`. We could add tests at both layers too, but I'd suggest writing the integration tests if we're only going to write one.

Reply:
Agreed. We can also test the permission layer that way.  I've been writing data source tests out of habit.  I'll fix it in a future PR.